.github/workflows/basic.yml

```diff
@@ -40,4 +40,4 @@ jobs:
       with:
         token: ${{ secrets.CODECOV_TOKEN }}
     - name: Test
-      run: go test -short -timeout 50m -v github.com/alibaba/opentelemetry-go-auto-instrumentation/test
+      run: go test -short -timeout 50m -v -run='.*Test[^Plugins|Latest|Muzzle].*' github.com/alibaba/opentelemetry-go-auto-instrumentation/test
```

Comment:
Is that means the ci only runs ut? Why not just remove the Test phase, there is a ut phase before the Test phase

Reply:
It runs `TestHelloworld,TestBuildProject` etc, please refer to https://github.com/alibaba/opentelemetry-go-auto-instrumentation/actions/runs/11569364635/job/32202935882?pr=142 for more details.